test/e2e/util/util.go

```diff
@@ -375,17 +375,17 @@ func EnsureAPIBudget(t *testing.T, ctx context.Context, client crclient.Client,
 		}{
 			{
 				name:   "control-plane-operator read",
-				query:  fmt.Sprintf("sum(hypershift:controlplane:component_api_requests_total{method=\"GET\", namespace=~\"%s\"}) by (pod)", namespace),
+				query:  fmt.Sprintf("sum(hypershift:controlplane:component_api_requests_total{app=\"control-plane-operator\", method=\"GET\", namespace=~\"%s\"}) by (pod)", namespace),
```

Comment:
Could we use backticks to not have to escape quotes so the query can easily be copied out?

Reply:
didn't know that was a thing :+1: